src/core/Microsoft.PowerToys.Settings.UI/Views/ImageResizerPage.xaml

```diff
@@ -40,86 +45,136 @@
+                                     Height="35" />
+
+                            <ComboBox 
+                                      SelectedIndex="{x:Bind Fit}"
+                                      Width="90" 
+                                      Height="35">
                                 <ComboBoxItem>Fill</ComboBoxItem>
                                 <ComboBoxItem>Fit</ComboBoxItem>
                                 <ComboBoxItem>Stretch</ComboBoxItem>
                             </ComboBox>
-                            <muxc:NumberBox Value="{x:Bind Width}"/>
-                            <TextBlock Text="x" FontWeight="SemiBold" Margin="0,4,0,0"/>
-                            <muxc:NumberBox Value="{x:Bind Height}"/>
-                            <ComboBox SelectedIndex="3">
+
+                            <muxc:NumberBox Value="{x:Bind Width}" 
+                                            Width="68" 
+                                            Height="35"/>
+
+                            <TextBlock Text="x" 
+                                       FontWeight="SemiBold" 
+                                       TextAlignment="Center" 
+                                       Margin="0,4,0,0" 
+                                       Width="25" 
+                                       Height="35"/>
+
+                            <muxc:NumberBox Value="{x:Bind Height}" 
+                                            Width="68" 
+                                            Height="35"/>
+
+                            <ComboBox SelectedIndex="{Binding Unit}"
+                                      Width="90" 
+                                      Height="35">
                                 <ComboBoxItem>Centimeters</ComboBoxItem>
                                 <ComboBoxItem>Inches</ComboBoxItem>
                                 <ComboBoxItem>Percent</ComboBoxItem>
-                                <ComboBoxItem>Pixels</ComboBoxItem>
+                                <ComboBoxItem >Pixels</ComboBoxItem>
                             </ComboBox>
-                            <Button x:Name="RemoveButton" Background="Transparent" Click="RemoveButton_Click">
-                                <Button.Content>
-                                    <FontIcon Opacity="0.8" Glyph="&#xE74D;"/>
-                                </Button.Content>
-                            </Button>
+                            <AppBarButton x:Name="RemoveButton" 
+                                          Background="Transparent" 
+                                          Command = "{Binding DeleteImageSizeEventHandler, Source={StaticResource ViewModel}}"
+                                          CommandParameter="{Binding Id}"                                          
+                                          Icon="Delete" 
+                                          Width="68" 
+                                          Height="35"/>
```

Comment:
There is a weird misalignment with all this UI elements.

Reply:
https://github.com/microsoft/PowerToys/pull/2324/commits/6b0ee21b54de0fac62c3d5250ecf143c4dcdfe0f

Reply:
I think this is the alignment issue he meant (I haven't checked the latest commits so I dont know if its already fixed)
![image](https://user-images.githubusercontent.com/32061677/80131637-5f89d300-854f-11ea-97ff-e42789fc3680.png)
The image sizes controls are a bit more away from the left edge than the other controls

Reply:
oh the padding sure, thanks.

Reply:
https://github.com/microsoft/PowerToys/pull/2324/commits/fa7741b0c2700eee5f1ed11e62c7a8f0b3a385d2